hubspot3/base.py

```diff
@@ -139,6 +140,11 @@ def _prepare_request(
         if data and headers["Content-Type"] == "application/json" and not retried:
             data = json.dumps(data)
 
+        i = 0
+        while i < len(properties):
+            url = url + "&properties=" + properties[i]
+            i += 1
+
```

Comment:
If we're using a list here after making the above changes, it's generally a bit more readable/pythonic to use a for loop in this case (also no need for a counter 😉):
```python
for property in properties:
    url += '&properties={}'.format(property)
```

I also normally prefer `.format()` to string concatenation, but it works either way.